cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -18,49 +18,21 @@ import (
 	"testing"
 	"time"
 
+	"github.com/hyperledger/aries-framework-go/pkg/restapi/helper"
+
```

Comment:
import order

Reply:
I moved the hyperledger import below the third party libraries. Was that what you were expecting?